thirdparty/s3.go

```diff
@@ -43,3 +46,28 @@ func PreSign(r RequestParams) (string, error) {
 	urlStr, err := req.Presign(PresignExpireTime)
 	return urlStr, err
 }
+
+func GetHeadObject(r RequestParams) (*s3.HeadObjectOutput, error) {
+	session, err := session.NewSession(&aws.Config{
+		Region: aws.String(endpoints.UsEast1RegionID),
+		Credentials: credentials.NewStaticCredentialsFromCreds(credentials.Value{
+			AccessKeyID:     r.AwsKey,
+			SecretAccessKey: r.AwsSecret,
+		}),
+	})
+	if err != nil {
+		return nil, err
+	}
+	svc := s3.New(session)
+
+	headObject, err := svc.HeadObject(&s3.HeadObjectInput{
+		Bucket: aws.String(r.Bucket),
+		Key:    aws.String(r.FileKey),
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return headObject, err
```

Comment:
This can just `return headObject, nil` since it's already known that the error is non-nil.